services/src/main/java/org/keycloak/authentication/authenticators/resetcred/ResetPassword.java

```diff
@@ -35,6 +38,7 @@ public void authenticate(AuthenticationFlowContext context) {
                 (context.getExecution().isConditional() &&
                         configuredFor(context))) {
             context.getAuthenticationSession().addRequiredAction(UserModel.RequiredAction.UPDATE_PASSWORD);
+            context.getAuthenticationSession().setClientNote(Constants.KC_ACTION_EXECUTING, UserModel.RequiredAction.UPDATE_PASSWORD.name());
```

Comment:
The problem with using `Constants.KC_ACTION_EXECUTING` is that it is related to Application Initiated Actions (AIA). It is a different way of executing required actions, not really as a step of the authentication flow.

I think we need something else to indicate to both the `UpdatePassword` required action and the `login-update-password.ftl` template to handle the option to logout sessions.

In fact, can we just have it enable all the time regardless if you are running the action as part of the authentication flow or as a AIA?

Reply:
If you mean to remove the if guard in the template I have no problem with that:
```
                        <#if isAppInitiatedAction??>
                            <div class="checkbox">
                                <label><input type="checkbox" id="logout-sessions" name="logout-sessions" value="on" checked> ${msg("logoutOtherSessions")}</label>
                            </div>
                        </#if>
```